### PR TITLE
dbt fixes for running external ClickHouse

### PR DESCRIPTION
**Pull request by bmtcril**

### CODE REVIEW

File: tutoraspects/templates/aspects/jobs/init/clickhouse/init-clickhouse.sh [hunk 1]
```diff
@@ -335,7 +335,7 @@ CREATE TABLE IF NOT EXISTS {{ ASPECTS_EVENT_SINK_DATABASE }}.{{ ASPECTS_EVENT_SI
 
 -- Grant permissions to the users
 GRANT INSERT, SELECT ON {{ ASPECTS_XAPI_DATABASE }}.* TO '{{ ASPECTS_CLICKHOUSE_LRS_USER }}';
-GRANT SELECT ON {{ ASPECTS_XAPI_DATABASE }}.* TO '{{ ASPECTS_CLICKHOUSE_REPORT_USER }}';
+GRANT SELECT, DROP TABLE, DROP VIEW ON {{ ASPECTS_XAPI_DATABASE }}.* TO '{{ ASPECTS_CLICKHOUSE_REPORT_USER }}';
```
Comment (bmtcril) [inline tutoraspects/templates/aspects/jobs/init/clickhouse/init-clickhouse.sh:338]: Non-select perms here can go away once @SoryRawyer creates a reporting schema for dbt to work in.

File: tutoraspects/templates/aspects/jobs/init/dbt/init-dbt.sh [hunk 2]
```diff
@@ -16,4 +16,4 @@ echo "Installing dbt dependencies"
 dbt deps --profiles-dir /app/aspects/scripts/aspects/
 
 echo "Running dbt"
-dbt run --profiles-dir /app/aspects/scripts/aspects/
+XAPI_SCHEMA={{ ASPECTS_XAPI_DATABASE }} ASPECTS_ENROLLMENT_EVENTS_TABLE={{ ASPECTS_ENROLLMENT_EVENTS_TABLE }} dbt run --profiles-dir /app/aspects/scripts/aspects/
```
Comment (bmtcril) [inline tutoraspects/templates/aspects/jobs/init/dbt/init-dbt.sh:19]: These changes aren't sustainable as sources grow, but work for now. We'll likely need a whole env vars file that we can source in or something.

**Review by Ian2012 — CHANGES_REQUESTED** [review]
just change that and we are good to go

**Review by SoryRawyer — COMMENTED** [review]
Ah, sorry I missed the `DBT_PROFILE_SECURE` variable. I agree that it'd be nice to have a somewhat-centrally-defined environment variable file, but I don't know enough about tutor to feel comfortable weighing in on where that should live/how it should be configured.

**Review by Ian2012 — COMMENTED** [review]
LGTM

**Comment by bmtcril** [comment]
@Ian2012 rebased on your latest changes, need a new thumb if it looks good

**Review by Ian2012 — APPROVED** [review]

**Pull request merged**